src/components/Input/Input.tsx

```diff
@@ -55,9 +52,11 @@ export interface BaseInputProps
   placeholder?: string;
   helperText?: string;
   toolTip?: string;
-  startAdornment?: React.ReactNode;
-  endAdornment?: React.ReactNode;
-  onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  disabled?: boolean;
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+    value?: number,
```

Comment:
Value ska bara vara number om type är number, annars ska den vara string.

Reply:
Reverted back to not using value